src/main/java/ca/warp7/frc2022/subsystems/Launcher.java

```diff
@@ -57,11 +72,10 @@ public void calcOutput() {
         if (targetRPS == 0.0)
             this.setVoltage(0.0);
         else
-            this.setVoltage((targetRPS + getError() * kLauncherKp) * kLauncherKv +
-                    kLauncherKs * Math.signum(targetRPS));
+            this.setVoltage(voltageCalculator.calculate(this.getError()));
```

Comment:
If you are using the kF term, then you should use the `calculate` method with _two_ arguments: one for the setpoint and one for the actual speed. If you just give it the error, the kF term is ignored